flink-end-to-end-tests/test-scripts/test_ha_per_job_cluster_datastream.sh

```diff
@@ -113,7 +113,6 @@ function run_ha_test() {
     # jm killing loop
     set_config_key "env.pid.dir" "${TEST_DATA_DIR}"
 
-    setup_flink_shaded_zookeeper ${ZOOKEEPER_VERSION}
```

Comment:
Is there a reason to keep `${ZOOKEEPER_VERSION}` as a parameter in this method? It's not really needed anymore analogously to the other files where we reverted mentioning the ZK version in the log output.

Reply:
We're even removing the ZK version in `run-nightly-tests.sh` as a parameter of `test_ha_per_job_cluster_datastream.sh`. So, I guess, we can clean it up entirely here.

Reply:
No I just missed it.